jenkins/helper/performance2json.py

```diff
@@ -159,12 +163,75 @@ def simple_performance_cluster(row):
                 "ms": 1000 * date.timestamp()
         }))
 
+def ddl_performance_cluster(rownum, row):
+        global version, current_date, branch, mode, edition, headline, values
+
+        if rownum == 0:
+                headline = row
+                for header in headline:
+                        values[header] = []
+        elif rownum == -1:
+                count = len(values[headline[0]])
+                number_runs = len(values[headline[0]])
+                date = current_date
+                size = 0
+
+                result = {
+                        "test": {
+                        },
+                        "size": {
+                                "collection": row[6],
+                                "count": count,
+                                "size": size
+                        },
```

Comment:
what is the meaning of size in this test?

Reply:
ok, this was still a question of mine to be asked - what should be put in there? the test parameters like collection count, index count, loop count?

Reply:
Yes, exactly.